app/views/events/index.html.erb

```diff
@@ -1,26 +1,43 @@
-<%= link_to 'New Event', new_event_path %>
-<h1>All Events</h1>
-<table border="1">
-  <tr>
-    <th>Logo</th>
-    <th>Title</th>
-    <th>Date</th>
-    <th>Time Start</th>
-    <th>Time End</th>
-  </tr>
-  <% @events.each do |event| %>
-    <tr>
-      <td><%= event.logo %></td>
-      <td><%= event.title %></td>
-      <td><%= event.date %></td>
-      <td><%= event.time_start %></td>
-      <td><%= event.time_end %></td>
+<div class="event">
+  <div class="container mt-3 p-0">
+    <div class="row justify-content-between col-min-width mt-5 pl-3 pr-3">
+      <span class="heading">Events</span>
+      <%= button_to 'Create Event', new_event_path, method: :get, class: "event-button bg-color" %>
+    </div>
+    <div class="row">
+
+      <% @events.each do |event| %>
+
+        <div class="col-md col-sm-12 col-xs col-min-width mx-3 event-card p-3">
```

Comment:
Can you let me know about using `col-md`, `col-xs`, `col-sm-12` that how it will behave with different kind of screen sizes?